File: python/006/main.py
# ProjectEuler 006
# Python / main.py
# copyright (c) 2015 Susisu
#
# Python 3.4.2
# `python3 main.py`


def main():
    s = 0
    t = 0
    for i in range(1, 101):
        s += i
        t += i * i
    print(s * s - t)

main()

File: julia/006/main.jl
# ProjectEuler 006
# Julia / main.jl
# copyright (c) 2015 Susisu
#
# Julia v0.4.1
# `julia main.jl`

function main()
    s = 0
    t = 0
    for i = 1:100
        s += i * i
        t += i
    end
    t = t * t
    println(t - s)
end

main()
